modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,310 @@
+----
+tinymce.init({
+  selector: "textarea",
+  plugins: [
+    "lists"
+  ],
+  toolbar: "bullist numlist",
+});
+----
+
 
-[[schema]]
-== Schema
+[[removed-plugins]]
+=== Removed Plugins
+
+In {productname} 7.0 the below plugin has been removed.
+
+[[removed-plugins-template-plugin]]
+==== Removed open-source `Template` plugin
+
+The open-source `Template` plugin and associated config options have been removed in {productname} 7.0.
+
+Customers using the `template` plugin need to migrate to the premium `advtemplate` which contains all the functionality except some esoteric options that was created very long ago. Instead of simply porting over these options, we will listen to customer feedback and quickly add the ones customer’s are relying on. 
+
+This plugin has been replaced with the **Advanced Template** plugin, see: xref:advanced-templates.adoc[Advanced Template] for more details.
+
+[IMPORTANT]
+For customers using the `template` plugin and are looking to migrate to the premium `advtemplate` plugin which contains all the functionality except some specific options which are noted below.
+
+Options not available in **Advanced Template** plugin
+    
+* `template_cdate_classes`
+* `template_cdate_format`
+* `template_mdate_classes`
+* `template_mdate_format`
+* `template_replace_values`
+* `template_preview_replace_values`
+* `template_selected_content_classes`
 
-...
 
 [[ui-and-ux-elements-and-components]]
 == UI and UX elements and components
 
-...
+=== Improved
+
+[[ui-and-ux-elements-and-components-improved-how-styles-are-applied-when-resizing-rows-and-columns-for-tables]]
+==== Improved how styles are applied when resizing rows and columns for Tables.
+
+Previously added numerous style elements when resizing table rows and columns such as on the `table element`, `tr elements`, and `td elements`. This resulted in unappealing HTML output.
+
+{productname} 7.0 addresses this, now, the height input field has been moved from the `cell` properties dialog to the `row` properties dialog. Now, when an existing table is resized using the drag handles or the row properties dialog, inline css height declarations will be be stripped from cells and table elements and only applied on rows.
+
+[NOTE]
+{productname} 7.0 does not provide any fallback to revert to the old behavior.
+
+=== Fixed
+
+[[ui-and-ux-elements-and-components-fixing-the-table-style-by-css-option]]
+==== Fixing the `table_style_by_css` option
+
+Previously, tables were styled using CSS rather than attributes, aligning with modern web development practices.
+
+However, due to {productname}'s legacy predating this shift, a default option called `table_style_by_css` was introduced in {productname} 6 to accommodate this transition. Unfortunately, the implementation of this feature resulted in an unexpected behavior where the older border attribute was not updated alongside CSS changes, causing issues such as the disappearance of dashed outlines around tables with no borders.
```

Comment:
`table_style_by_css` was available in TInyMCE 5 so this paragraph isn't accurate

Reply:
fixed in our review